azerothcore/azerothcore-wotlk
# Core/Account: Add total time played in account

##### CHANGES PROPOSED:
-  Add total time played for account


##### ISSUES ADDRESSED: 
- No


##### TESTS PERFORMED:
- [x] Tested in game
- [x] Tested build
- [x] Tested OS Windows

##### HOW TO TEST THE CHANGES:
- Apply test commit
- To stay in the game for a while.
- Quit the game completely
- See the value in the database auth.account - totaltime

##### KNOWN ISSUES AND TODO LIST:
- [x] Test OS Unix

##### Target branch(es):
- Master
